src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -68,6 +68,18 @@ else
    exit
 endif
 
+#######################################################
+#  if LADAS_COUPLING ==2, do atmens forcing processing 
+###################################################### 
+   if ( $LADAS_COUPLING == 2) then
+     cd $HOMDIR
+     module load  python/GEOSpyD/Ana2019.03_py3.7
+     module  load nco/4.8.1
```

Comment:
@weiyuan-jiang :
We also load nco in process_rst.csh (where we can probably clean up the if block that was needed for the SLES11-to-SLES12 transition period).
Is there a better way to loading these modules through the environment? 
Perhaps nco isn't by default in the g5 environment, but how about python?

Reply:
I think at this point, nco is already loaded, so it is not necessary here.